src/main/java/com/exadel/discount/repository/TestUserRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.exadel.discount.repository;
+
+import com.exadel.discount.entity.Role;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+
+@Repository
+public class TestUserRepository {
+    public static com.exadel.discount.repository.User findByEmail(String email) {
+        ArrayList<com.exadel.discount.repository.User> list = new ArrayList<>();
+        list.add(new com.exadel.discount.repository.User(
```

Comment:
for that, you could create a Liquibase script. 
If it's for testing purposes it shouldn't be in the main directory.

Reply:
better use Liquibase